src/app/team/page.tsx

```diff
@@ -0,0 +1,301 @@
+  );
+}
+
+function ActivityStatusToggle({ filter }: { filter?: ActivityStatus }) {
+  const t = useTranslations();
+  const dispatch = useAppDispatch();
+  const { activityStatusFilter } = useAppSelector((state) => state.teamSlice);
+  const label = filter === undefined ? "all" : filter;
+  const isActive = activityStatusFilter === filter;
+  return (
+    <div
+      className={`cursor-pointer inline-block mx-1 mt-3 px-4 py-2 bg-[#232629] opacity-60 rounded-badge hover:opacity-100 ${
+        isActive ? "!opacity-100 bg-dexter-green text-black" : ""
+      }`}
+      onClick={() =>
+        dispatch(teamSlice.actions.setActivityStatusFilter(filter))
+      }
+    >
+      <p className="text-sm font-bold">{t(label)}</p>
+    </div>
+  );
+}
+
+function ExpertiseToggle({ filter }: { filter?: Expertise }) {
+  const t = useTranslations();
+  const dispatch = useAppDispatch();
+  const { expertiseFilter } = useAppSelector((state) => state.teamSlice);
+  const label = filter === undefined ? "all" : filter;
+  const isActive = expertiseFilter === filter;
+  return (
+    <div
+      className={`cursor-pointer inline-block mx-1 mt-3 px-4 py-2 bg-[#232629] opacity-60 rounded-badge hover:opacity-100 ${
+        isActive ? "!opacity-100 bg-dexter-green text-black" : ""
+      }`}
+      onClick={() => dispatch(teamSlice.actions.setExpertiseFilter(filter))}
+    >
+      <p className="text-sm font-bold">{t(label)}</p>
+    </div>
+  );
+}
+
+function Contributors() {
+  const t = useTranslations();
+  const { teamSlice } = useAppSelector((state) => state);
+  const contributors = selectFilteredContributors(teamSlice);
+  return (
+    <div className="my-10">
+      {teamSlice.isLoading ? (
+        <>
+          <div className="flex justify-center">
+            <SkeletonRectangle width="w-40" height="h-6" />
+          </div>
+          <div className="flex flex-wrap justify-center">
+            {[1, 2, 3, 4, 5, 6, 7, 8].map((_, indx) => {
+              return (
+                <SkeletonRectangle
+                  width="w-[250px]"
+                  height="h-[120px]"
+                  key={indx}
+                  className="m-2 bg-[#2C2C2C]"
+                />
+              );
+            })}
+          </div>
+        </>
+      ) : (
+        <>
+          <p className="text-base text-center opacity-70">
+            {t("N_contributors_found").replaceAll(
+              "<$NUMBER>",
+              contributors.length.toString()
+            )}
+          </p>
+          <div className="flex flex-wrap justify-center">
+            {contributors.map((contributor, indx) => {
+              return <ContributorCard contributor={contributor} key={indx} />;
+            })}
+          </div>
+        </>
+      )}
+    </div>
+  );
+}
+
+function ContributorCard({ contributor }: { contributor: Contributor }) {
+  const t = useTranslations();
+  return (
+    <div className="w-[250px] h-[120px] bg-[#232629] rounded-2xl m-2 p-4 relative">
```

Comment:
Can we use bg-base-100 instead of 232629

Reply:
fixed!